pdr_backend/accuracy/app.py

```diff
@@ -350,12 +354,40 @@ def serve_statistics_from_file():
     If the file cannot be read or another error occurs, it returns a 500 Internal Server Error.
     """
 
+    four_weeks_ago = datetime.utcnow() - timedelta(weeks=4)
+    start_ts = UnixTimeS(int(four_weeks_ago.timestamp()))
+    end_ts = UnixTimeS(int(datetime.utcnow().timestamp()))
     try:
-        with open(JSON_FILE_PATH, "r") as f:
-            data = json.load(f)
-            response = jsonify(data)
-            response.headers.add("Access-Control-Allow-Origin", "*")  # Allow any origin
-            return response
+        db_conn = DuckDBDataStore("./lake_data", read_only=True)
+        slots_table_name = Slot.get_lake_table_name()
+        slots_table = db_conn.query_data(
+            f"""
+            SELECT * FROM {slots_table_name} WHERE SLOT > {start_ts} AND SLOT < {end_ts}
+            """
+        )
+        db_conn.duckdb_conn.close()
+
+        slots_table = slots_table.group_by("ID").first()
+        slots_table = slots_table.sort("slot")
+
+        all_slots: List[Slot] = []
+
+        # Iterate over rows and create objects
+        for row in slots_table.rows(named=True):
```

Comment:
please see what I did here with ArgFeeds https://github.com/oceanprotocol/pdr-backend/blob/42d25e9fefe86358446a2bbe892fd8a658639533/pdr_backend/cli/arg_feeds.py#L82

I think you can add a function in Slot called from_table_row. I want this to become the standard where each object that has a database representation also knows how to load itself from its corresponding database row. And, in the future, for all lists of objects to support loading from a table or database result. It's useful for other instances, so please move this part into the Slots class itself.

Reply:
Same as before. I agree.

I have documented this in [#1424](https://github.com/oceanprotocol/pdr-backend/issues/1424)